css/styles.css

```diff
@@ -0,0 +1,116 @@
+    
+    header{
+
+        text-align:left;
+        height:20%;
+        padding-left:30px;
+        display:inline-block;
+
+    }
+
+    photos{
+
+        height:150px;
+        width:830px;
+        display:inline-block;
+        padding-top: 10px;
+        padding-bottom:10px;
+        padding-left:30px;
+        padding-right:10px;
+        margin-bottom:30px;
+        margin-left:30px;
+        clear:left;
+        clear:right;
+
+    }
+
+    article{
+
+        height:185px;
+        font-family: 'Neuton',serif;
+        font-size:18px;
+    }
+
+
+    h1{
+        color:pink;
+        font-size: 48px;
+        font-family: 'Bungee Shade', cursive;
+         
+    }
+
+    h2{
+
+        color:lightblue;
+        font-size:36;
+        font-family:'Space Mono',monospace;
+        font-weight: #400;
+        font-style:italic;
+    }
+
+    h3{
+
+        color:green
```

Comment:
This is missing a semi-colon. All of the code below here will not run until this is added. 🔨